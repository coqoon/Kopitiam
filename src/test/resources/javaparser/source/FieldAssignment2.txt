class Foo {
  int b;
  void set () {
    b = b + 10;
  }
}
