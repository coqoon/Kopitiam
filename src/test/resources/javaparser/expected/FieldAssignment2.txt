class Foo {
  int b;
  void set () {
    int tmp_1 = this.b;
    this.b = tmp_1 + 10;
  }
}
